src/components/Breadcrumb.svelte

```diff
@@ -1,31 +1,65 @@
 <script>
   export let links;
+
+  const isPlatform = (app) => {
+    const platforms = ["ios", "android", "fenix", "amazon", "echo", "fire_tv"];
+    const match = platforms.filter((item) => app.includes(item));
+    if (match.length) return true;
+    return false;
+  };
+
+  const getPlatformLogo = (app) => {
+    if (app.includes("ios")) return "/img/app-logos/apple-breadcrumb.png";
+    if (app.includes("fenix") || app.includes("android"))
+      return "/img/app-logos/android-breadcrumb.png";
+    if (
+      app.includes("amazon") ||
+      app.includes("echo") ||
+      app.includes("fire_tv")
+    )
+      return "/img/app-logos/amazon-breadcrumb.png";
+    return undefined;
+  };
 </script>
 
-<ol>
-  {#each links as link}
-    <li>
-      <a class="link-name" href={link.url}>{link.name} </a><span class
-        >{links.indexOf(link) === links.length - 1 ? "" : "/"}</span
-      >
-    </li>
-  {/each}
-</ol>
+<div class="breadcrumb">
+  <ol>
+    {#each links as link}
+      <li>
+        <a class="link-name" href={link.url}>{link.name} </a>
+        <span>{links.indexOf(link) === links.length - 1 ? "" : "->"}</span>
```

Comment:
I changed to using the arrow "-->" instead of "/" here because I find it to look slightly cleaner. Happy to revert it though if it's better to use the original.

Reply:
I like it! Let's keep it.